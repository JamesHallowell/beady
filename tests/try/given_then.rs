use beady::scenario;

#[scenario]
fn given_then_is_invalid() {
    'given_something: {
        'then_something: {
            assert!(true);
        }
    }
}

fn main() {}
