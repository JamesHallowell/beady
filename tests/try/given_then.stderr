error: Only "when" or "and_given" are allowed inside "given" section
 --> tests/try/given_then.rs:6:9
  |
6 |         'then_something: {
  |         ^^^^^^^^^^^^^^^
